guilford_dps_offenders.yaml

```diff
@@ -0,0 +1,330 @@
+        column: "OFFENDER GENDER CODE"
+        transforms:
+        - !<transforms.ReplaceTransform>
+          target: ["MALE", "FEMALE"]
+          goal: ["M", "F"]
+      nc.PersonBirthDate:
+        type: "nc.PersonBirthDate"
+        column: "OFFENDER BIRTH DATE"
+        transforms:
+        - !<transforms.DateTransform>
+          pattern: ["yyyy-MM-dd HH:mm:ss", "yyyy-MM-dd HH:mm:ss.S"]
+      nc.PersonRace:
+        type: "nc.PersonRace"
+        # column: "OFFENDER RACE CODE"
+        transforms:
+        - !<transforms.BooleanRegexTransform>
+          column: "OFFENDER RACE CODE"
+          pattern: "(\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?)"
+          transformsIfTrue:
+          - !<transforms.ValueTransform>
+            value: null
+          transformsIfFalse:
+          - !<transforms.ColumnTransform>
+            column: "OFFENDER RACE CODE"
+          - !<transforms.ReplaceTransform>
+            target: [ "BLACK", "UNKNOWN", "OTHER", "ASIAN/ORTL", "INDIAN", "WHITE"]
+            goal: ["black", "", "other", "asian", "amindian", "white"]
+      nc.PersonRace2:
+        type: "nc.PersonRace"
+        # column: "OFFENDER ETHNIC CODE"
+        transforms:
+        - !<transforms.BooleanRegexTransform>
+          column: "OFFENDER ETHNIC CODE"
+          pattern: "(\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?)"
+          transformsIfTrue:
+          - !<transforms.ValueTransform>
+            value: null
+          transformsIfFalse:
+          - !<transforms.ColumnTransform>
+            column: "OFFENDER ETHNIC CODE"
+          - !<transforms.ReplaceTransform>
+            target: ["EUROPEAN/N.AM./AUSTR", "AMERICAN INDIAN", "SLAVIC (E. EUROPEAN)", "ORIENTAL", "OTHER", "ASIAN", "UNKNOWN", "HISPANIC/LATINO", "AFRICAN", "NORDIC/SCANDANAVIAN", "PACIFIC ISLANDER"]
+            goal: ["white", "amindian", "white", "asian", "other", "asian", "", "", "black", "white", "pacisland" ]
+      nc.PersonEthnicity:
+        type: "nc.PersonEthnicity"
+        # column: "OFFENDER ETHNIC CODE"
+        transforms:
+        - !<transforms.BooleanRegexTransform>
+          column: "OFFENDER ETHNIC CODE"
+          pattern: "(\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?)"
+          transformsIfTrue:
+          - !<transforms.ValueTransform>
+            value: null
+          transformsIfFalse:
+          - !<transforms.ColumnTransform>
+            column: "OFFENDER ETHNIC CODE"
+          - !<transforms.ReplaceTransform>
+            target: ["EUROPEAN/N.AM./AUSTR", "AMERICAN INDIAN", "SLAVIC (E. EUROPEAN)", "ORIENTAL", "OTHER", "ASIAN", "UNKNOWN", "HISPANIC/LATINO", "AFRICAN", "NORDIC/SCANDANAVIAN", "PACIFIC ISLANDER"]
+            goal: ["nonhispanic", "nonhispanic", "nonhispanic", "nonhispanic", "nonhispanic", "nonhispanic", "", "hispanic", "nonhispanic", "nonhispanic", "nonhispanic"]
```

Comment:
Isn't it more accurate to leave everything blank except for the hispanic/latino entry? For the others we don't really know if they are hispanic or not - one can be any of those races but also hispanic.

Reply:
that's true, i'll change it to blank